docs/Basics/index.md

```diff
@@ -6,37 +6,38 @@ The variety of AEC (Architecture, Engineering and Construction) commercial softw
+In such a large sector, most problems you encounter are likely to have been solved before by someone in your organisation, or outside. What frequently happens is that the wheel gets reinvented. Additionally, when a script is created, it often exists in isolation, and is used only by a small group of people. 
 
 Let's take the example of a user wanting to create some functionality in an Excel macro. This has several issues:
 
 - **Maintanability issues**. Frequently, only the original creator of a Macro knows how to develop or maintain it. If they are not around and a problem arises, the Macro is often just thrown away.
 
-- **Shareability issues**. For example, if the creator of a Macro that does some function in a spreadsheet wants to share this functionality with one of their clients, they end up sharing the spreadsheet with the Macro in it, effectively ending up sharing the source code, fruit of their hard work, which could end up being copied.
+- **Shareability issues**. For example, if the creator of a Macro that performs some function in a spreadsheet wants to share this functionality with one of their clients, they often end up sharing the spreadsheet with the Macro in it, effectively sharing the source code and the fruit of their hard work, which ends up just being copied.
 
 - **Scalability issues**. Macros are hard to scale. If you need to add more features or more complex functionality in a Macro, the code soon becomes unmanageable.
 
-- **Collaboration issues**. Collaborating to the code written in a Macro is a mess. Only one person can work on it at the same time; in order to merge the work of different people into a same Macro spreadsheet, either a library is created, or copy-paste is required.
+- **Collaboration issues**. Collaborating on the code written in a Macro is a mess. Only one person can work on it at the same time. In order to merge the work of different people into a same Macro spreadsheet, either a library is created, or copy-paste is required.
 
-BHoM proposes to create a common library of functionality in **Engine**s, which are simply tidy collections of functions. Like Adapters, Engines can be stored in Toolkits, which are simply GitHub repositories. GitHub repositories make it easy to share and collaborate on code (or not share, if privacy is chosen). BHoM makes it very easy to expose any functionality written in an Engine to Excel, Grasshopper or other interfaces.
+BHoM proposes to create a common library of functionality in **Engines**, which are simply tidy collections of functions. Like Adapters, Engines can be stored in Toolkits, which are simply GitHub repositories. GitHub repositories make it easy to share and collaborate on code (or not share, if privacy is chosen). BHoM makes it very easy to expose any functionality written in an Engine to Excel, Grasshopper or any other interface.
```

Comment:
`Engine` is a key word within BHoM terminology, which is why it's made bold without the `s` which pluralises the key word.

`other interfaces` plural is cleaner than `any other interface`, however, the lack of a comma after `Grasshopper` makes the sentence run on a bit - so suggesting adding a comma.

```suggestion
BHoM proposes to create a common library of functionality in **Engine**s, which are simply tidy collections of functions. Like Adapters, Engines can be stored in Toolkits, which are simply GitHub repositories. GitHub repositories make it easy to share and collaborate on code (or not share, if privacy is chosen). BHoM makes it very easy to expose any functionality written in an Engine to Excel, Grasshopper, or other interfaces.
```